nsqadmin/logger.go

```diff
@@ -1,5 +1,58 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
```

Comment:
I had a similar thought, but additionally was going to suggest that we make levels a type that can be shared across applications. We can provide a function that takes a string and returns an appropriately typed value and also implement the `Stringer` interface on the type to satisfy the purpose here.

I _think_ it should also be possible to cast an int directly to this type if its signature is `type LogLevel int`?